docs/build/blast-install-1.md

```diff
@@ -39,31 +29,13 @@ Checkout [Setup Rust](/docs/build/setup-rust)
 
 ## 02 Install Cudos Blast
 
-### Globally - Create a new project anywhere
-
-Use `npm` to install Cudos Blast CLI tool globally. This means you can create a new project anywhere. 
-
-```bash
-npm install cudos-blast -g
-
-# confirm installation
-
-blast --version
-```
-:::info TIP
-
-Use `npx cudos-blast` to run commands. This ensures you are always using the latest version. 
-:::
-
-### Locally - Add to a single project
-
-If you only want to install Cudos Blast into a single project then do the following:
+Install Cudos Blast into a 'counter' project as follows:
```

Comment:
I don't think we should include these instructions.

Either way, these instructions don't end up in the situation in "Create Counter project".

For that, you need to run `blast init`, not `npm init`. The blast init command creates the project structure that is explained by Create Counter project.

You need to do this whether or not you install cudos-blast globally!

My recommendation is to run `blast init` once to create your global node location. Starting the node for the first time creates important configuration files in the current directory, which are an inconvenience to replace if they go missing. So that shouldn't happen in a tutorial directory you might throw away.

(This should be fixed by fixing Cudos Blast so it doesn't store part of its global configuration in a random location.)